apps/src/redux/scriptSelectionRedux.js

```diff
@@ -73,6 +73,26 @@ export const getSelectedScriptName = state => {
   return scriptName;
 };
 
+/* Get the user friendly name of a script(the unit or course name) */
+export const getSelectedScriptFriendlyName = state => {
+  const scriptId = state.scriptSelection.scriptId;
+  if (!scriptId) {
+    return null;
+  }
+
+  const scripts = state.scriptSelection.validScripts;
+  let scriptName = null;
+  for (let i = 0; i < scripts.length; i++) {
+    const script = scripts[i];
+    if (script.id === scriptId) {
+      scriptName = script.name;
+      break;
+    }
+  }
```

Comment:
this can be simplified a bit using standard libraries:
```js
const script = scripts.find(script => script.id === scriptId)
return script ? script.name : null;
```

Reply:
Does this just replace the for loop?